experiments/AMIP/coupler_driver.jl

```diff
@@ -713,8 +717,8 @@ if ClimaComms.iamroot(comms_ctx)
         )
     end
 
-    ## sample animations
-    if !is_distributed && config_dict["anim"]
+    ## sample animations (not compatible with MPI)
+    if !CA.is_distributed(ClimaComms.context(boundary_space)) && config_dict["anim"]
```

Comment:
Don't use have a `comms_ctx` variable already (instead of fetching it from the boundary space)?